package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "react-native-kontaktio",
-  "version": "2.10.0",
+  "version": "3.0.0",
```

Comment:
I think this version bump should be done in a distinct PR when publishing a new version of the lib.

Reply:
Exactly! Together with an updated [Changelog](https://github.com/Driversnote-Dev/react-native-kontaktio/blob/1c15200818f161ac7531e9295fdf6decbaba661d/CHANGELOG.md)